server/events/policy_check_output_generator.go

```diff
@@ -0,0 +1,113 @@
+package events
+
+import (
+	"fmt"
+
+	"github.com/runatlantis/atlantis/server/events/command"
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/lyft/feature"
+)
+
+const KeySeparator = "||"
+
+type PolicyCheckOutputStore struct {
+	Store map[string]*models.PolicyCheckSuccess
+}
+
+func NewPolicyCheckOutputStore() *PolicyCheckOutputStore {
+	return &PolicyCheckOutputStore{
+		Store: map[string]*models.PolicyCheckSuccess{},
+	}
+}
+
+func (p *PolicyCheckOutputStore) buildKey(projectName string, workspace string) string {
+	return fmt.Sprintf("%s%s%s", projectName, KeySeparator, workspace)
+}
+
+func (p *PolicyCheckOutputStore) GetOutputFor(projectName string, workspace string) *models.PolicyCheckSuccess {
+	key := p.buildKey(projectName, workspace)
+
+	if output, ok := p.Store[key]; ok {
+		return output
+	}
+	return nil
+}
+
+func (p *PolicyCheckOutputStore) WriteOutputFor(projectName string, workspace string, policyCheckResult command.ProjectResult) {
```

Comment:
Just have the generator initialize the store with this data, Seems pointless to have an exported method for this.